src/ray/common/id.h

```diff
@@ -78,78 +86,175 @@ class UniqueID : public BaseID<UniqueID> {
 
 class JobID : public BaseID<JobID> {
  public:
-  static constexpr int64_t length = 4;
+  static constexpr int64_t LENGTH = 4;
 
-  // TODO(qwang): Use `uint32_t` to store the data.
   static JobID FromInt(uint32_t value);
 
-  static size_t Size() { return length; }
+  static size_t Size() { return LENGTH; }
 
   static JobID FromRandom() = delete;
 
   JobID() : BaseID() {}
 
  private:
-  uint8_t id_[length];
+  uint8_t id_[LENGTH];
+};
+
+class ActorID : public BaseID<ActorID> {
+ private:
+  static constexpr size_t UNIQUE_BYTES_LENGTH = 4;
```

Comment:
The naming convention here needs to change, it should be kUniqueBytesLength, see https://google.github.io/styleguide/cppguide.html#Constant_Names

Reply:
Thanks for your information. :)